Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py

```diff
@@ -118,12 +118,11 @@ def http_request(url, method, headers=None, body=None, params=None, files=None):
 
     if result.headers['Content-Type'] == 'application/octet-stream':
         return result
-    else:
-        try:
-            json_res = json.loads(xml2json(result.text))
-            return json_res
-        except Exception:
-            return_error(f'Failed to parse response to json. response: {result.text}')
+    try:
+        json_res = json.loads(xml2json(result.text))
+        return json_res
+    except Exception:
+        return_error(f'Failed to parse response to json. response: {result.text}')
```

Comment:
we should try to avoid using `return_error` in the code, but only in main try/except. Instead we should use raise